elementary/monitor/data_monitoring/data_monitoring.py

```diff
@@ -123,17 +123,31 @@ def get_latest_invocation(self) -> Dict[str, Any]:
             return {}
 
     @staticmethod
-    def _check_dbt_package_compatibility(dbt_pkg_ver: str):
+    def _check_dbt_package_compatibility(dbt_pkg_ver: str) -> None:
         dbt_pkg_ver = version.parse(dbt_pkg_ver)
         py_pkg_ver = version.parse(package.get_package_version())
-        logger.info(
-            f"Checking compatibility between edr ({py_pkg_ver}) and Elementary's dbt package ({dbt_pkg_ver})."
-        )
-        if (
-            dbt_pkg_ver.major != py_pkg_ver.major
-            or dbt_pkg_ver.minor != py_pkg_ver.minor
+        if dbt_pkg_ver.major > py_pkg_ver.major or (
+            dbt_pkg_ver.major == py_pkg_ver.major
+            and dbt_pkg_ver.minor > py_pkg_ver.minor
+        ):
+            logger.warning(
+                f"You are using incompatible versions between edr ({py_pkg_ver}) and Elementary's dbt package ({dbt_pkg_ver}).\n "
+                "To fix please run:\n"
+                "pip install elementary-data --upgrade\n",
+            )
+            return
+
+        if dbt_pkg_ver.major < py_pkg_ver.major or (
+            dbt_pkg_ver.major == py_pkg_ver.major
+            and dbt_pkg_ver.minor < py_pkg_ver.minor
         ):
             logger.warning(
                 f"You are using incompatible versions between edr ({py_pkg_ver}) and Elementary's dbt package ({dbt_pkg_ver}).\n "
-                "Please upgrade the major and minor versions to align.\n",
+                "To fix please update your packages.yml, and run:\n"
+                "dbt deps && dbt run --select elementary\n",
             )
+            return
+
+        logger.info(
```

Comment:
Maybe we should just drop this log?
I tend to think that it's more confusing to the user rather than valuable information.

Reply:
I thought about this, but it could help us when debugging problems for users. 
Also, I think that when they run the tool it's overall positive that they see progress and don't think it's stuck.